core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJob.java

```diff
@@ -57,36 +61,39 @@ protected void executeTracedJob(JobExecutionContext context) throws JobExecution
         Stack stack = stackService.getByIdWithListsInTransaction(getStackId());
         Status stackStatus = stack.getStatus();
         if (!Status.getUnschedulableStatuses().contains(stackStatus)) {
-            applyStackPatches(stack);
+            String stackPatchTypeName = context.getJobDetail().getJobDataMap().getString(STACK_PATCH_TYPE_NAME);
+            StackPatchType stackPatchType = stackPatchTypeConverter.convertToEntityAttribute(stackPatchTypeName);
+            if (StackPatchType.UNKNOWN.equals(stackPatchType)) {
+                throw new JobExecutionException(String.format("Stack patch type %s converted to UNKNOWN", stackPatchTypeName));
```

Comment:
would this result in unscheduling the `STACK_PATCH_TYPE_NAME` from running into this again and again until a new version is released?